activerecord/README.rdoc

```diff
@@ -195,6 +195,35 @@ Source code can be downloaded as part of the Rails project on GitHub:
 * https://github.com/rails/rails/tree/master/activerecord
 
 
+== Getting Started
+
+1.install sqlite to your PC
+
+Here is example for Mac User. 
+Please use suitable install command for your environment.
+
+  $ brew install sqlite
+
+
+2.install ActiveRecord gem
+  
+  $ gem install activerecord
+
+3.install sqlite3 gem
+
+  $ gem install sqlite3
+
+4.prepare simple.rb locally
+https://github.com/rails/rails/blob/master/activerecord/examples/simple.rb
```

Comment:
Sorry for vague sentence. 
I mean simply creating the simple.rb. 
I updated content.